Sources/Valet/SecureEnclaveAccessControl.swift

```diff
@@ -24,19 +24,17 @@ import Foundation
 @objc(VALSecureEnclaveAccessControl)
 public enum SecureEnclaveAccessControl: Int, CustomStringConvertible, Equatable {
```

Comment:
We couldn't use `CaseIterable` here due to the `@available` tags below. I think using availability here is better than the comments we had before. (We couldn't do this before due to Xcode 9 support)